src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx

```diff
@@ -32,14 +47,21 @@ const socialLinksData: ISocialLinkData[] = [
 ];
 
 const SocialLinks = () => (
-  <FlexRow justify="flex-end">
-    {socialLinksData.map(({ component: Component, label, href }, index) => (
-      <FlexCol xs="auto" key={`footer-social-link-${index}`}>
-        <FooterLink href={href} aria-label={label}>
-          <Component size="30px" />
-        </FooterLink>
-      </FlexCol>
-    ))}
+  <FlexRow justify="space-between">
+    <FlexCol xs={12} s={6} m={4} l={2}>
+      <FooterLink href="https://www.zopa.com">
+        <ZopaLogo />
+      </FooterLink>
+    </FlexCol>
+    <FlexCol xs={12} s={6} m={4} l={3}>
+      <SFlexRow justify="flex-end" gutter={24}>
+        {socialLinksData.map(({ component: Component, label, href }, index) => (
+          <SFooterLink href={href} aria-label={label} key={`footer-social-link-${index}`}>
```

Comment:
Minor ( _doesn't block merge_ 😄 )  

I believe using array indexes as `key` is not recommended, we could use the `href` as the unique value instead?
```jsx
<SFooterLink  key={`footer-social-link-${href}`}>
```

Reply:
👍  done